feature/history/src/commonMain/kotlin/org/mifospay/feature/history/HistoryScreen.kt

```diff
@@ -75,54 +76,61 @@ internal fun HistoryScreenContent(
     showTopBar: Boolean = true,
     onBackClick: (() -> Unit)? = null,
 ) {
-    Column(
+    MifosScaffold(
         modifier = modifier.fillMaxSize(),
-    ) {
-        if (showTopBar && onBackClick != null) {
-            MifosTopBar(
-                topBarTitle = stringResource(Res.string.feature_history_title),
-                backPress = onBackClick,
-            )
-        }
-
-        Box(
-            modifier = Modifier
-                .fillMaxSize()
-                .weight(1f),
-            contentAlignment = Alignment.Center,
-        ) {
-            when (state.viewState) {
-                is HistoryState.ViewState.Loading -> {
+        topBar = if (showTopBar && onBackClick != null) {
+            {
+                MifosTopBar(
+                    topBarTitle = stringResource(Res.string.feature_history_title),
+                    backPress = onBackClick,
+                )
+            }
+        } else {
+            {}
```

Comment:
don't need to have else block just have a look it looks idle. use let or like below which will be empty if the condition doesn't pass.
```
topBar = {
            if (showTopBar && onBackClick != null) {
                MifosTopBar(
                    topBarTitle = "",
                    backPress = {},
                )
            }
        }
```